test/utils.test.js

```diff
@@ -4,9 +4,6 @@ beforeEach(() => jest.resetModules());
 
 test('bundleFeeds() throws error when bundling JS', async () => {
     expect.hasAssertions();
-    jest.doMock('@asset-pipe/js-reader', () => () =>
-        Promise.reject(new Error())
-    );
     const { bundleFeeds, endWorkers } = require('../lib/utils');
```

Comment:
does this `require` still have to be inline?

Reply:
unsure, will test out.

Reply:
most likely yes since i needed to keep `beforeEach(() => jest.resetModules());`

Reply:
why? if we don't mock anymore

Reply:
yup, seems so

Reply:
because new Worker is called at the to of the utils module and multiple calls to endWorker break things

Reply:
i think it probably should have all been inside a class or closure